app/helpers/application_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
   def full_title page_title
-    base_title = It18n.t "base_title"
+    base_title = t "base_title"
```

Comment:
Sao lại xóa `I18n.` thế, model với helper phải ghi đủ nhé

Reply:
ghi đủ no báo lỗi ko chạy chị

Reply:
haiz, `I18n` chứ, sao có `It` kìa

Reply:
em sửa up lại r mà chị